platform/os_linux.go

```diff
@@ -53,34 +54,46 @@ func GetOSInfo() string {
 }
 
 func GetProcessSupport() error {
```

Comment:
I assume the function that are suing their own client are a part of CNS or other packages only and not cni ?

Reply:
This one is not used anywhere actually, not sure whether we should remove it